### PR TITLE
Adding new phantomjs pre-req and requirejs packages

### PR DESCRIPTION
**Pull request by urfolomeus**
Whenever I install RAD fresh I get a `missing requirejs` error, so I've explicitly added it to the project manifest. Also PhantomJS was complain about a missing pre-req, so I added that too.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,6 +24,8 @@
     "karma-spec-reporter": "0.0.13",
     "mocha": "~1.15.1",
     "phantomjs": ">= 1.9.7-1",
+    "phantomjs-prebuilt": ">= 1.9",
```
Comment (munckymagik) [inline package.json]: Can we go for Phantom >=2? Seems like this might be the moment to try?

Comment (urfolomeus) [inline package.json]: It seems that we can. :)

Not only that but it removes the requirement to manually add the `-prebuilt` package too. \o/

**Comment by urfolomeus** [comment]
@munckymagik OK it would seem that the best way to do this is to upgrade the karma-phantomjs-runner instead and have it install phantomjs. Need to manually add the -prebuilt though.

**Comment by Tr4pSt3R** [comment]
:+1:

**Comment by munckymagik** [comment]
Checked out, installed and tests run fine. :+1:

**Pull request merged**